### DISCUSSION TITLE
Team meeting tomorrow!

### DISCUSSION THREAD

**Discussion by jeremymanning** [ContextLab/chatify#8] [Announcements]
We'll have a team meeting tomorrow (10:30AM -- 11:30AM EST) on Zoom (link is in the [slack workspace](https://online-summer-school.slack.com/archives/C050Y7E4DA4)).  Agenda:
  - Talk through basic project organization and [open issues](https://github.com/ContextLab/chatify/issues)
  - Ideally assign specific people or teams to specific tasks
  - Update from @HemuManju about the "base" version of the package (what it does, how it works, etc.)

**Comment by jeremymanning** [comment]
Hi again!  We'll have another group meeting on Thursday May 4 (10:30 -- 11:30AM EST) using the same zoom link as last week (link is on slack; see above post).  Agenda for this week:
  - check ins about ongoing [tasks/issues](https://github.com/ContextLab/chatify/issues)
  - open general discussion
  - plan out the next week + self-assign tasks

If anyone would like to add something to the agenda, just comment here or on slack!

**Reply by HemuManju** [reply]
I have been developing an IPython widget that builds upon the existing package. I can discuss the implementation at the meeting tomorrow. Looking forward to receiving feedback from the group!

**Comment by jeremymanning** [comment]
Hi Team,

Our next team meeting is will be on Tuesday 6/20 (9:30 -- 10:30AM EST) using the zoom link posted on our slack workspace.  Agenda:

- Summary of current project status
- Check-ins about how the example notebooks @HemuManju put together are working
- Discussion about the possibility of setting up a server to handle queries
- Outline remaining tasks and approximate timeline

**Comment by jeremymanning** [comment]
Hey Team!

Our next team meeting is tomorrow (July 6) at 10:00AM EST, using the zoom link posted in our slack workspace.  Agenda:
- Summary of current project status + updates from @HemuManju 
- Discussion of remaining tasks and issues before we're ready to release an MVP
- Decision about whether to attempt to get an MVP ready for this year's neuromatch, or to wait until next year

I'll record the meeting and post a link to the recording in slack for those who can't attend live.  And if you'd like to add anything to the agenda, just comment here or on slack as usual!

See you tomorrow 😄!
